beacon-chain/blockchain/chain_info_test.go

```diff
@@ -27,6 +29,48 @@ var _ ChainInfoFetcher = (*Service)(nil)
 var _ TimeFetcher = (*Service)(nil)
 var _ ForkFetcher = (*Service)(nil)
 
+// setupInsertParameters prepares a beacon State with the given data to mock
+// insert into forkchoice
+func setupInsertParameters(
```

Comment:
Too generic of a function name. I'd prefer something like `prepareForkchoiceState` or `buildForkchoiceState`